taxonomies/categories.txt

```diff
@@ -658,6 +658,30 @@ hu:Almaborok
 nl:Ambachtelijke ciders
 ro:Cidru
 
+<en:Ciders
+en:Traditional Welsh Cider
+protected_name_file_number_en: PGI-GB-01251
```

Comment:
search and replace all protected_name_file_number_en: by protected_name_file_number:en:

Reply:
fixé